internal/provider/serverless_cluster.go

```diff
@@ -54,17 +56,57 @@ func ResourceServerlessCluster() *schema.Resource {
 				// always returns a non-nil error.
 				// ForceNew: true,
 				Type: schema.TypeString,
+				// TODO: Make ForceNew to work
+				// ForceNew:    true,
 			},
 			"free_tier": {
 				Description: "Whether a cluster is in a free tier",
-				Computed:    true,
+				Default:     true,
+				Type:        schema.TypeBool,
+				Optional:    true,
+				// TODO: Make ForceNew to work
+				// ForceNew:    true,
+			},
+			"enable_dns": {
+				Description: "Whether to enable CNAME for seed nodes",
+				Optional:    true,
 				Type:        schema.TypeBool,
+				// TODO: Make ForceNew to work
+				// ForceNew:    true,
+				Default: true,
+			},
+			"units": {
+				Description: "Processing units",
+				Default:     1,
+				Type:        schema.TypeInt,
+				Optional:    true,
+				// TODO: Make ForceNew to work
+				// ForceNew:    true,
+			},
+			"hours": {
+				Description: "Hours for cluster to last",
+				Type:        schema.TypeInt,
+				Optional:    true,
+				// TODO: Make ForceNew to work
+				// ForceNew:    true,
 			},
 		},
 	}
 }
 
 func resourceServerlessClusterCreate(ctx context.Context, d *schema.ResourceData, meta interface{}) diag.Diagnostics {
+	freeTier := d.Get("free_tier")
```

Comment:
d.GetOk - please use here

Reply:
what for ? get will return value respecting `default` from the config

Reply:
previously the code was before like:

```
freeTier := d.Get("free_tier")

if freeTier.(bool) {
```

I misread it and assumed your are testing for interface{} nil value. If you use default, you can do type conversion right away:

```
freeTier := d.Get("free_tier").(bool)
```